src/System.Data.Common/ref/System.Data.Common.csproj

```diff
@@ -4,11 +4,11 @@
   <PropertyGroup>
     <AssemblyVersion>4.0.1.0</AssemblyVersion>
     <OutputType>Library</OutputType>
-    <NuGetTargetMoniker>.NETStandard,Version=v1.0</NuGetTargetMoniker>
+    <NuGetTargetMoniker>.NETStandard,Version=v1.2</NuGetTargetMoniker>
```

Comment:
Let's keep this one targeting netstandard1.0 same as the impl and add a coment about doing so in order to support win8/wp8.